shashi/feb10.js

```diff
@@ -0,0 +1,37 @@
+// Write a function that creates an object with each (key, value) pair being the (lower case, upper case) versions of a letter, respectively.
+// Examples
+// mapping(["p", "s"]) ➞ { "p":"P", "s":"S" }
+// mapping(["a", "b", "c"]) ➞ { "a": "A", "b": "B", "c": "C" }
+// mapping(["a", "v", "y", "z"]) ➞ { "a": "A", "v": "V", "y": "Y", "z": "Z"}
+//Assumption: If any value of input array is not string, same value is added to key and value
```

Comment:
hmm this wouldnt work if the value is a function or another array/object. Suggest to not add to the obj